### PR TITLE
chore: describe data sources

### PR DESCRIPTION
**Pull request by rix0rrr**
This information needs to be documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
...
+  be also rendered in the future, but are not right now. In a model diff, type history is rendered with a `⇐` between them, in order
+  from new to old. If you see `type1 ⇐ type2 ⇐ type3`, then `type3` is the oldest and is the one that will be used by CDK.
+* **Other property attributes are overwritten**: things like documentation, optionality, etc. are overwritten, so newer specifications
+  fully overwrite the information from older specifications.
+
+Sources are read in this order:
+
+| What | Description | Updates |
+|------|-------------|--------------------
+| Resource Spec | This is the original [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html), which is being replaced by the Registry Schema. Imported in order from `us-east-1`, `us-west-2`. | Frozen at version `144.0.0` (Oct 13, 2023). |
```
Comment (otaviomacedo) [inline README.md:32]: Does this refer to the files in the [cfnspec/spec-source/specification/000_cfn/000_official][1] folder? I can see version 130.1.0 there.

[1]: https://github.com/aws/aws-cdk/tree/main/packages/%40aws-cdk/cfnspec/spec-source/specification/000_cfn/000_official

File: README.md [hunk 2]
```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
 - `@aws-cdk/service-spec-types` - loading, saving, querying and diffing a db
 - `@aws-cdk/aws-service-spec` - the actual db file
 
+## Data Sources
```
Comment (otaviomacedo) [inline README.md:14]: Do we not use patches anymore?

File: README.md (hunk 1 above)
Comment (rix0rrr) [reply README.md:32]: It does not. It's the copy in this repository.

File: README.md (hunk 2 above)
Comment (rix0rrr) [reply README.md:14]: Fair enough.

Comment (mrgrain) [reply README.md:14]: > Do we not use patches anymore?

Yes and no. But we need to describe that. 😅

File: README.md (hunk 1 above)
Comment (mrgrain) [reply README.md:32]: The spec in `aws/aws-cdk` is about to go away. Will need a week or two to complete that work.

File: README.md [hunk 3]
```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
...
+
+| What | Description | Updates |
+|------|-------------|--------------------
+| Resource Spec | This is the original [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html), which is being replaced by the Registry Schema. Imported in order from `us-east-1`, `us-west-2`. | Frozen at version `144.0.0` (Oct 13, 2023). |
+| SAM Resource Spec | This is the unofficial SAM resource spec as voluntarily maintained by the [GoFormation](https://github.com/awslabs/goformation) project | Updated daily | 
+| Registry Schema | This is the new [CloudFormation Resource Provider Schema](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html), replacing the old Resource Spec. It is more expressive than the old spec. Imported in order: `us-east-1`, `us-east-2`, `us-west-2`. | Updated daily |
+| SAM JSON Schema | This is the newer version of the unofficial [GoFormation](https://github.com/awslabs/goformation) SAM specification, expressed in JSON Schema. | Updated daily |
+| CloudFormation Docs | A JSON rendering of the AWS CloudFormation Resource Reference. | Updated weekly |
```
Comment (mrgrain) [inline README.md:36]:
```suggestion
| CloudFormation Docs | A JSON rendering of the AWS CloudFormation Resource Reference. | Weekly |
```

File: README.md [hunk 4]
```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
...
+  fully overwrite the information from older specifications.
+
+Sources are read in this order:
+
+| What | Description | Updates |
+|------|-------------|--------------------
+| Resource Spec | This is the original [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html), which is being replaced by the Registry Schema. Imported in order from `us-east-1`, `us-west-2`. | Frozen at version `144.0.0` (Oct 13, 2023). |
+| SAM Resource Spec | This is the unofficial SAM resource spec as voluntarily maintained by the [GoFormation](https://github.com/awslabs/goformation) project | Updated daily | 
+| Registry Schema | This is the new [CloudFormation Resource Provider Schema](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html), replacing the old Resource Spec. It is more expressive than the old spec. Imported in order: `us-east-1`, `us-east-2`, `us-west-2`. | Updated daily |
+| SAM JSON Schema | This is the newer version of the unofficial [GoFormation](https://github.com/awslabs/goformation) SAM specification, expressed in JSON Schema. | Updated daily |
```
Comment (mrgrain) [inline README.md:35]:
```suggestion
| SAM JSON Schema | This is the newer version of the unofficial [GoFormation](https://github.com/awslabs/goformation) SAM specification, expressed in JSON Schema. | Daily |
```

File: README.md [hunk 5]
```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
...
+  from new to old. If you see `type1 ⇐ type2 ⇐ type3`, then `type3` is the oldest and is the one that will be used by CDK.
+* **Other property attributes are overwritten**: things like documentation, optionality, etc. are overwritten, so newer specifications
+  fully overwrite the information from older specifications.
+
+Sources are read in this order:
+
+| What | Description | Updates |
+|------|-------------|--------------------
+| Resource Spec | This is the original [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html), which is being replaced by the Registry Schema. Imported in order from `us-east-1`, `us-west-2`. | Frozen at version `144.0.0` (Oct 13, 2023). |
+| SAM Resource Spec | This is the unofficial SAM resource spec as voluntarily maintained by the [GoFormation](https://github.com/awslabs/goformation) project | Updated daily |
```
Comment (mrgrain) [inline README.md:33]:
```suggestion
| SAM Resource Spec | This is the unofficial SAM resource spec as voluntarily maintained by the [GoFormation](https://github.com/awslabs/goformation) project | Daily | 
```

File: README.md [hunk 6]
```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
...
+* **Other property attributes are overwritten**: things like documentation, optionality, etc. are overwritten, so newer specifications
+  fully overwrite the information from older specifications.
+
+Sources are read in this order:
+
+| What | Description | Updates |
+|------|-------------|--------------------
+| Resource Spec | This is the original [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html), which is being replaced by the Registry Schema. Imported in order from `us-east-1`, `us-west-2`. | Frozen at version `144.0.0` (Oct 13, 2023). |
+| SAM Resource Spec | This is the unofficial SAM resource spec as voluntarily maintained by the [GoFormation](https://github.com/awslabs/goformation) project | Daily | 
+| Registry Schema | This is the new [CloudFormation Resource Provider Schema](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html), replacing the old Resource Spec. It is more expressive than the old spec. Imported in order: `us-east-1`, `us-east-2`, `us-west-2`. | Updated daily |
```
Comment (mrgrain) [inline README.md:34]:
```suggestion
| Registry Schema | This is the new [CloudFormation Resource Provider Schema](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html), replacing the old Resource Spec. It is more expressive than the old spec. Imported in order: `us-east-1`, `us-east-2`, `us-west-2`. | Daily |
```

**Review by otaviomacedo — APPROVED** [review]

**Pull request merged**